src/llv8.h

```diff
@@ -403,6 +403,8 @@ class LLV8 {
 
   void Load(lldb::SBTarget target);
 
+  constants::Types types;
```

Comment:
Hm... I see it now. We should move `GetTypeName` to the `Value` class then. Thank you!